polliwog/polyline/_polyline_object.py

```diff
@@ -614,3 +614,38 @@ def sectioned(self, section_breakpoints, copy_vs=False):
             Polyline(v=maybe_copy(self.v[start:end]), is_closed=False)
             for (start, end) in zip(section_starts, section_ends)
         ]
+
+    def point_along_curve(self, percentage_along_curve):
+        """
+        Selects a point on a polyline, the given percentage along the cuve.  Ex:
+        Halfway.
+
+        Args:
+            percentage_along_curve: floating point value between 0 and 1.
+
+        Returns:
+            A point on the polyline that is the given percentage distance
+            from the starting point to the endpoint.
+        """
```

Comment:
Some initial tweaks, feel free to adjust further:

```suggestion
    def point_along_path(self, fraction_of_total):
        """
        Selects a point the given fraction of the total length of the polyline. For
        example, to find the halfway point, pass `fraction_of_total=0.5`.

        Args:
            fraction_of_total (float): Fraction of the total length, from 0 to 1

        Returns:
            A point on the polyline that is the given fraction of the total length
            from the starting point to the endpoint.
        """
```